src/shims/shims.proj

```diff
@@ -36,30 +36,36 @@
           Outputs="$(GenFacadesSemaphoreFile)">
 
     <PropertyGroup>
-      <!--<GenFacadesArgs>$(GenFacadesArgs) -contracts:"@(GenFacadesContracts, ',')"</GenFacadesArgs>-->
-      <GenFacadesArgs>$(GenFacadesArgs) -seeds:"@(GenFacadesSeeds, ',')"</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -facadePath:"$(GenFacadesOutputPath)"</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -producePdb:false</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -assemblyFileVersion:$(AssemblyFileVersion)</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -forceZeroVersionSeeds</GenFacadesArgs>
-      <!-- TODO: We should remove this flag once we have all the types for netstandard -->
-      <GenFacadesArgs>$(GenFacadesArgs) -ignoreMissingTypes</GenFacadesArgs>
+      <ProducePdb>false</ProducePdb>
+      <GenFacadesIgnoreMissingTypes>true</GenFacadesIgnoreMissingTypes>
+      <GenFacadesIgnoreBuildAndRevisionMismatch>false</GenFacadesIgnoreBuildAndRevisionMismatch>
+      <GenFacadesForceZeroVersionSeeds>true</GenFacadesForceZeroVersionSeeds>
+      <GenFacadesBuildPartialReferenceFacade>false</GenFacadesBuildPartialReferenceFacade>
     </PropertyGroup>
 
-    <MakeDir Directories="$(IntermediateOutputPath)" />
-    <WriteLinesToFile File="$(GenFacadesResponseFile)" Lines="$(GenFacadesArgs)" Overwrite="true" />
-
-    <PropertyGroup>
-      <GenFacadesCmd>$(ToolHostCmd) "$(ToolsDir)GenFacades.exe"</GenFacadesCmd>
-    </PropertyGroup>
-
-    <Exec Condition="'@(NetFxContracts)' != ''"
-          Command="$(GenFacadesCmd) -contracts:&quot;@(NetFxContracts)&quot; @&quot;$(GenFacadesResponseFile)&quot;"
-          WorkingDirectory="$(ToolRuntimePath)" />
-
-    <Exec Condition="'@(NETStandardContracts)' != ''"
-          Command="$(GenFacadesCmd) -contracts:&quot;@(NETStandardContracts)&quot; @&quot;$(GenFacadesResponseFile)&quot;"
-          WorkingDirectory="$(ToolRuntimePath)" />
+    <GenFacadesTask
+      Condition="'@(NetFxContracts)' != ''"
+      Contracts="@(NetFxContracts)"
+      Seeds="@(GenFacadesSeeds, ',')"
```

Comment:
I don't think the `',')` is necessary, similar below. Feel free to clean up after merging.